frontend/src/pages/TermPlanner/optionsHeader/main.jsx

```diff
@@ -68,55 +68,61 @@ const OptionsHeader = ({
-            style={{ width: "200px" }}
-            okText="Yes"
-            cancelText="No"
-          >
-            <Tooltip title="Unplan all courses">
-              <button type="button" className="settings-button">
-                <FaRegCalendarTimes className="settings-icon" />
+          <div>
+            <Popconfirm
+              placement="bottomRight"
+              title="Are you sure you want to unplan all your courses?"
+              onConfirm={() => dispatch(unscheduleAll())}
+              style={{ width: "200px" }}
+              okText="Yes"
+              cancelText="No"
+            >
+              <Tooltip title="Unplan all courses">
+                <button type="button" className="settings-button">
+                  <FaRegCalendarTimes className="settings-icon" />
+                </button>
+              </Tooltip>
+            </Popconfirm>
+          </div>
+        )}
+
+        {areYearsHidden && (
+          <div>
+            <Tooltip title="Show all hidden years">
+              <button type="button" className="settings-button" onClick={() => dispatch(unhideAllYears())}>
+                <EyeFilled className="settings-icon" />
               </button>
             </Tooltip>
-          </Popconfirm>
+          </div>
         )}
 
-        {areYearsHidden && (
-          <Tooltip title="Show all hidden years">
-            <button type="button" className="settings-button" onClick={() => dispatch(unhideAllYears())}>
-              <EyeFilled className="settings-icon" />
+        <div>
+          <Tooltip title="Toggle warnings for previous terms">
+            <button
+              className={`settings-button ${suppress ? "filled" : ""}`}
+              type="button"
+              onClick={() => {
+                setSuppress((prev) => !prev);
+                updateAllWarnings(
+                  dispatch,
+                  { years, startYear, completedTerms },
+                  { programCode, specialisation, minor },
+                  suppress,
+                );
+              }}
+            >
+              <WarningFilled className="settings-icon" />
             </button>
           </Tooltip>
-        )}
-        <Tooltip title="Toggle warnings for previous terms">
-          <button
-            className={`settings-button ${suppress ? "filled" : ""}`}
-            type="button"
-            onClick={() => {
-              setSuppress((prev) => !prev);
-              updateAllWarnings(
-                dispatch,
-                { years, startYear, completedTerms },
-                { programCode, specialisation, minor },
-                suppress,
-              );
-            }}
-          >
-            <WarningFilled className="settings-icon" />
-          </button>
-        </Tooltip>
+        </div>
       </div>
 
       <Tippy
         content={<HelpMenu />}
         moveTransition="transform 0.2s ease-out"
-        interactive
+        interactive="true"
```

Comment:
lol does tippy require you to pass true as a string? thats so troll

Reply:
oops don't think i need that there ahaha